### PR TITLE
O3-2241: Use pre-filled docker images for running e2e tests

### PR DESCRIPTION
**Pull request by Piumal1999**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

*None.*

## Related Issue

https://issues.openmrs.org/browse/O3-2241

## Other

*None.*

### CODE REVIEW

File: e2e/support/github/docker-compose.yml [hunk 1]
```diff
@@ -15,35 +16,9 @@ services:
       API_URL: /openmrs
 
   backend:
-    image: openmrs/openmrs-reference-application-3-backend:${TAG:-nightly}
+    image: piumal1999/openmrs-reference-application-3-backend:nightly-with-data #TODO: replace with OMRS docker image
```
Comment (Piumal1999) [inline e2e/support/github/docker-compose.yml:19]:
```suggestion
    image: openmrs/openmrs-reference-application-3-backend:nightly-with-data
```

File: e2e/support/github/docker-compose.yml [hunk 2]
```diff
@@ -15,35 +16,9 @@ services:
...
-      MYSQL_USER: ${OPENMRS_DB_USER:-openmrs}
-      MYSQL_PASSWORD: ${OPENMRS_DB_PASSWORD:-openmrs}
-      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-openmrs}
-    volumes:
-      - db-data:/var/lib/mysql
-
-volumes:
-  openmrs-data: ~
-  db-data: ~
+    image: piumal1999/openmrs-reference-application-3-db:nightly-with-data #TODO: replace with OMRS docker image
```
Comment (Piumal1999) [inline e2e/support/github/docker-compose.yml:24]:
```suggestion
    image: openmrs/openmrs-reference-application-3-db:nightly-with-data
```

Comment (Piumal1999) [inline e2e/support/github/docker-compose.yml:24]:
```suggestion
    image: openmrs/openmrs-reference-application-3-db:nightly-with-data
```

**Review by jayasanka-sack — APPROVED** [review]
This is awesommme!!! Thanks, @Piumal1999 !! 🎉

**Pull request merged**